api/app/views.py

```diff
@@ -47,7 +47,7 @@ def project_overrides(request):
         "fpr": "FIRST_PROMOTER_ID",
         "headway": "HEADWAY_API_KEY",
         "hideInviteLinks": "DISABLE_INVITE_LINKS",
-        "linkedin_api_key": "LINKEDIN_API_KEY",
+        "linkedinPartnerTracking": "LINKEDIN_PARTNER_TRACKING",
```

Comment:
I see here that we've replaced `linkedin_api_key` but not removed it from the settings [here](https://github.com/Flagsmith/flagsmith/pull/3880) - just checking this was intentional?

Reply:
Yeah it isnt used anywhere by the looks of it

Reply:
linkedin_api_key doesnt seem to be used anywhere else unless I'm missing something

Reply:
I understand that this API configuration is only needed for self-hosted deployments, right? Are we interested in having this tracking enabled for them?

Reply:
I'd say it's as good as any place to put it regardless of that, maybe we should have a wider conversation around our env vars, permanent flags and project_prod.js at some point.